resources/istio/charts/customization/values.yaml

```diff
@@ -0,0 +1,3 @@
+---
+enabled: false
+namespaces2Label: []
```

Comment:
I would put the values here, it will be easier to upgrade charts (just copy paste sub chart)